manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     disableProjects: false
     disablePipelines: false
     disableModelServing: false
-    disableProjectScoped: false
+    disableProjectScoped: true
```

Comment:
You have to remove this line if you're not releasing

Reply:
Ah yes, missed that. Thanks